src/Engine/BasicEngine.php

```diff
@@ -42,6 +42,15 @@ public function escapeLike(string $parameter): string
         return str_replace(['%', '_'], ['\\%', '\\_'], $parameter);
     }
 
+    public function getSqlParamValue($value): string
```

Comment:
I would rather this be called `exportParameter`, since it not a getter and could modify the input.

Reply:
Seems logical. Changed.